kotlin/gradle.properties

```diff
@@ -15,6 +15,7 @@
 #
 org.gradle.jvmargs='-Dfile.encoding=UTF-8'
 org.gradle.parallel=true
+android.enableJetifier=true
```

Comment:
Something is still bringing in support annotations, this fixes that.

Reply:
This might also fix the androidx test rule for fixing the main dispatcher if we try to use it again: I noticed that JetBrains is using a pre-AndroidX jar for some of their integration, which I think was hitting the wrong assertMainThread.